src/components/ExerciseIcons.tsx

```diff
@@ -0,0 +1,104 @@
+import styled from 'styled-components';
+import { colors, mixins } from '../../src/theme';
+import { ExerciseData } from '../../src/types';
+import * as Icons from './Icons';
+
+type ExplanationType = 'multiple-choice' | 'two-step';
+
+const TypeExplanations: { [key in ExplanationType]: string } = {
+  'multiple-choice': 'Select the best answer from the given list of distractors. Your instructor may or may not allow multiple attempts.',
+  'two-step': 'In a two-step question, OpenStax Tutor asks for your own answer first, then gives multiple-choice options to help you assess your learnings. \
+Recalling the answer to a question from memory helps you to retain things longer. Later, you and your instructor can review your work together.',
+}
+
+const ItemWrapper = styled.div`
+  padding: 0.6rem 0.9rem;
+  .popover { display: none; }
+  &:hover {
+    svg path { fill: ${colors.palette.mediumBlue};}
+    .popover { display: flex; }
+  }
+`;
+
+const InnerWrapper = styled.div`
+  position: relative;
+  display: flex;
+`;
+
+const Popover = styled.div`
+  ${mixins.popover}
+  top: 3rem;
+  width: 20rem;
+  position: absolute;
+  font-size: 1.2rem;
+  line-height: 1.8rem;
+`;
+
+interface PopoverItemProps {
+  text: string;
+  children: JSX.Element;
+  wrapperProps?: {
+    as: keyof JSX.IntrinsicElements;
+    href?: string;
+    target?: string;
+  }
```

Comment:
I don't love typing it like this but I failed at figuring out the right generic and gave up 😬